### PR TITLE
Page indicator

### PR DESCRIPTION
**Pull request by danielpeng2**
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,4 +51,4 @@
     "webpack-dev-server": "^3.2.1",
     "webpack-node-externals": "^1.7.2"
   }
-}
+}
```
Comment (Dendrimer) [inline package.json]: new line

File: src/components/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import logo from "../logo.png";
```
Comment (Dendrimer) [inline src/components/Header.jsx]: New line here between react imports and our imports

File: src/containers/PageIndicatorContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { connect } from "react-redux";
+import PageIndicatorComponent from "../components/PageIndicator";
```
Comment (Dendrimer) [inline src/containers/PageIndicatorContainer.jsx:3]: new line between our imports and the react imports

File: src/App.js [hunk 4]
```diff
@@ -1,20 +1,55 @@
...
-import Questionaire from './components/Questionaire.jsx';
+import Header from "./components/Header";
+import CounterContainer from "./containers/CounterContainer";
+// import PageIndicatorContainer from "./containers/PageIndicatorContainer";
+=======
+import { BrowserRouter as Router, Route } from "react-router-dom";
+
+import Intro from "./components/Intro.jsx";
+import Questionaire from "./components/Questionaire.jsx";
+>>>>>>> 03248c2b7dbd83f1adbe5b77e99313240df0894f
```
Comment (Dendrimer) [inline src/App.js]: This is broked

**Review by Dendrimer — APPROVED** [review]
Approving cause we ran out of time

**Review by YangStephen — APPROVED** [review]
Approved due to end of term merge.

**Pull request merged**